configHelper/tests/resources/sampleJenkinsOICAuth.yaml

```diff
@@ -0,0 +1,256 @@
+jenkins:
+  log:
+    recorders:
+      - loggers:
+          - level: "FINE"
+            name: "org.jenkinsci.plugins.workflow.job.WorkflowRun"
+        name: "workflowRun"
+  agentProtocols:
+    - "JNLP4-connect"
+    - "Ping"
+  authorizationStrategy: "loggedInUsersCanDoAnything"
+  crumbIssuer:
+    standard:
+      excludeClientIPFromCrumb: false
+  disableRememberMe: false
+  labelAtoms:
+    - name: "built-in"
+    - name: "main-node"
+  labelString: "main-node"
+  markupFormatter:
+    rawHtml:
+      disableSyntaxHighlighting: true
+  mode: EXCLUSIVE
+  myViewsTabBar: "standard"
+  numExecutors: 4
+  primaryView:
+    all:
+      name: "all"
+  projectNamingStrategy: "standard"
+  quietPeriod: 5
+  scmCheckoutRetryCount: 0
+  securityRealm:
+    oic:
```

Comment:
Not sure why they wrote `oic` instead of `oidc` in the config, how do you think this should be handled while adding the config. Let user add `oic` instead of `oidc` or keep the `oidc` norm and let the code handle the conversion?

Reply:
I think from my perspective this is pretty abstracted away from users and what we have should be fine. The user would only be providing values within this `oic` block and unless a user is digging around their Jenkins yaml they they shouldn't be aware, and even then it should be a small dot to connect

Reply:
I think I see what you meant now, my preference is keep the oidc norm and let code handle the conversion